W-16026636 Making fixes (#341)
mulesoft/docs-access-management: modules/ROOT/pages/multi-factor-authentication.adoc

Multi-factor authentication (MFA) provides an additional layer of verification for Anypoint Platform users.
MuleSoft requires MFA when users access Anypoint Platform, either directly with their Anypoint credentials or via single sign-on (SSO).

> MFA is enabled for direct signins and can’t be turned off at the organization level. However, there are a few service account types that don’t require MFA. An organization administrator should waive MFA for these accounts to allow existing integrations to continue without interruption.

Reviewer: ```suggestion
MFA is enabled for direct signins and can’t be turned off at the organization level. However, a few service account types don’t require MFA. An organization administrator should waive MFA for these accounts so that existing integrations can continue without interruption.
```

== Multi-Factor Authentication Requirements for Salesforce and MuleSoft Products

* Effective February 1, 2022, MFA must be enabled for all of your Anypoint Platform users. To learn more about this contractual requirement, see the https://help.salesforce.com/apex/HTViewSolution?urlname=Salesforce-Multi-Factor-Authentication-FAQ[Salesforce Multi-Factor Authentication FAQ]

> * As of October 29, 2022, all non-SSO accounts that do not have MFA configured are prompted to register for an MFA verification method and use that method each time they sign in to Anypoint Platform.

Reviewer: ```suggestion
* As of October 29, 2022, all non-SSO accounts that do not have MFA configured are prompted to register for an MFA verification method and use it to sign in to Anypoint Platform.
```
